src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,171 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static final int EMPTY_LIST = 0;
+
+    private Node<T> head;
+
+    private Node<T> tail;
```

Comment:
Don't use class Objects. Consider making `Node<T>` an inner class to ensure it's tied to the `MyLinkedList` class and not used elsewhere inappropriately.